app/views/layouts/angular/_multi_auth_credentials.html.haml

```diff
@@ -486,6 +562,7 @@
 %div{"ng-if" => "#{ng_model}.ems_controller == 'ems_container'"}
   :javascript
     miq_tabs_show_hide("#metrics_tab", true);
+    miq_tabs_show_hide("#virtualization_tab", true);
```

Comment:
there should also be: `miq_tabs_show_hide("#alerts_tab", true);`
but maybe not in this PR

Reply:
@zeari added in this PR.